src/main/io/rcdevice_cam.c

```diff
@@ -231,26 +227,23 @@ static void rcdevice5KeySimulationProcess(timeUs_t currentTimeUs)
     }
 #endif
 
-    if (camDevice->serialPort == 0) {
+    if (ARMING_FLAG(ARMED)) {
```

Comment:
It seems you've removed a required check for the serial port, which looks like it could be NULL at this point (`rcdevice5KeySimulationProcess()` is called unconditionally from `rcdeviceUpdate()`).